runtime/src/main.rs

```diff
@@ -109,15 +95,7 @@ extern "C" fn nmi_handler(trap_record: &TrapRecord) {
         err_interrupt_status,
     );
 
-    {
-        let soc_ifc = soc_ifc.regs_mut();
-        let ext_info = soc_ifc.cptra_fw_extended_error_info();
-        ext_info.at(0).write(|_| trap_record.mcause);
-        ext_info.at(1).write(|_| trap_record.mscause);
-        ext_info.at(2).write(|_| trap_record.mepc);
-        ext_info.at(3).write(|_| trap_record.ra);
-        ext_info.at(4).write(|_| err_interrupt_status);
```

Comment:
Would be good to keep the interrupt status. Could pass an `Option<u32>` to `log_trap_record` since the exception handler doesn't populate this value

Reply:
done